client/dist/styles/bubble.css

```diff
@@ -0,0 +1,17 @@
+#style-wrapper {
+    display: grid;
+    grid-template-columns: 145px 145px 145px 145px;    
+}
+
```

Comment:
good choice of style selectors.
The paint time of the page is somewhat dependent on this choice and specificity of IDs and classes is quicker than, for example, using universal or parent selectors , is much better for render speed